tests/cuda_ndarrays/conftest.py

```diff
@@ -0,0 +1,132 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+# pylint: disable=arguments-differ, inconsistent-return-statements, protected-access, abstract-method/
+import subprocess
+import os
+import pathlib
+import sys
+import shutil
+import pytest
+
+NEEDS_FROM_PARENT = hasattr(pytest.Item, "from_parent")

```

Comment:
What does this do? Does it depend on the pylint version?

Reply:
It is checking if `pytest.Item` class has the `from_parent` attribute. The purpose is to ensure the compatibility with different versions of pytest

Reply:
Have you found versions of pytest which don't have the attribute?

Reply:
Actually I don't see this attribute in the documentation so maybe my question is rather where have you seen this attribute?

Reply:
(maybe the answer is in the `ndarrays` folder?)

Reply:
[this](https://docs.pytest.org/en/7.1.x/deprecations.html#fspath-argument-for-node-constructors-replaced-with-pathlib-path) shows that it was deprecated and I think I added it for it to be backward compatible.

Reply:
Ok. That is fine with me :+1: